src/main/proto/protoServer.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+option java_multiple_files = true;
+package grpc.demoApp;
+
+service protoServer {
+  rpc getStockByItemId(GetStockByItemIdRequest) returns(GetStockByItemIdResponse);
+  rpc getItemsInStock(Empty) returns(stream Items);
+  rpc updateItemsInStock(stream UpdateItemsInStockRequest) returns(stream Item);
+}
+
+message GetStockByItemIdRequest {
+  int64 id = 1;
+}
+
+message GetStockByItemIdResponse {
+  int32 stock = 1;
+}
+
+message Items {
+  repeated Item items = 1;
+}
+
+message UpdateItemsInStockRequest {
+  int64 id = 1;
+  string operation = 2;
```

Comment:
"deduction" and "addition" will be two mutually exclusive fields. So only one of them is required in each request. 
e.g.
{
    id: 1,
    deduction: 1
}
OR
{
    id: 1,
    addition: 1
}

Reply:
I have a new idea today which I think might be better. 
Do you think it's better to have an Enumeration type which can be set to "deduction" or "addition, and one more Integer field for value?

Reply:
Also I modified the grammar a little bit in a way which I think better for future extension.
https://github.com/parasoft/parasoft-demo-app/pull/252/commits/dfc1c7fb509ef21f1bcecb59ca59a7e1df5773c7